README.md

```diff
@@ -1,21 +1,25 @@
 jsapi-resources
 ===============
-A collection of resources for developers using the [ArcGIS API for JavaScript](https://js.arcgis.com).
+A collection of resources for developers using the [ArcGIS API for JavaScript](https://js.arcgis.com) to create local builds. For more information, visit the [Developer Tooling](https://developers.arcgis.com/javascript/latest/guide/tooling-intro/) guide topic in the SDK.
 
-**ArcGIS API for JavaScript 3.x**
+#### ArcGIS API for JavaScript 3.x
 * [Bower - Create custom builds of the API](./3.x/bower/README.md)
 * [TypeScript - Class and interface definitions](./3.x/typescript/README.md)
 
-**ArcGIS API for JavaScript 4.x**
+#### ArcGIS API for JavaScript 4.x
+
+##### ES modules (ESM)
+* [ESM samples - Create local builds with frameworks and module bundlers](./esm-samples/)
+
+##### AMD modules
 * [Webpack - Create custom builds of the API](./4.x/webpack/README.md)
-* [AMD Builds - Create local builds of the API](./4.x/amd/README.md)
+* [Dojo and RequireJS - Create local builds of the API](./4.x/amd/README.md)
 * [TypeScript - Class and interface definitions](./4.x/typescript/README.md)
-* [Working with Frameworks](./frameworks/README.md)
 
-**Coding Standards**
+#### Coding Standards
 * [JSHint options for the JS API](./jshint/README.md)
```

Comment:
Should we remove JSHint since it hasn't been updated in six years and we've moved to ESLint?

Reply:
I wondered about that as well. I'm okay to remove

Reply:
@dasa have we posted any eslint info that we can share here?

Reply:
No, not that I know of.